src/Chainweb/Miner/Coordinator.hs

```diff
@@ -20,71 +22,89 @@
 --
 
 module Chainweb.Miner.Coordinator
-  ( -- * Types
-    MiningState(..)
-  , MiningStats(..)
-  , PrevTime(..)
-  , ChainChoice(..)
-  , PrimedWork(..)
-  , MinerStatus(..), minerStatus
-    -- * Functions
-  , newWork
-  , publish
-  ) where
-
-import Data.Aeson (ToJSON)
-import Data.Bool (bool)
+( -- * Types
+  MiningState(..)
+, MiningStats(..)
+, PrevTime(..)
+, ChainChoice(..)
+, PrimedWork(..)
+, MinerStatus(..), minerStatus
+  -- * Functions
+, newWork
+, publish
+) where
```

Comment:
A lot of the logic for creating a new block headers and extending a cut got moved from this module to `Chainweb.Cut.Create`, where all functionality for extending a cut is centralized, now.